controllers/state_of_the_world.go

```diff
@@ -0,0 +1,132 @@
+package controllers
+
+import (
+	"context"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/kuadrant/policy-machinery/controller"
+	"github.com/kuadrant/policy-machinery/machinery"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	ctrlruntime "sigs.k8s.io/controller-runtime"
+	gwapiv1 "sigs.k8s.io/gateway-api/apis/v1"
+
+	kuadrantv1alpha1 "github.com/kuadrant/kuadrant-operator/api/v1alpha1"
+	kuadrantv1beta1 "github.com/kuadrant/kuadrant-operator/api/v1beta1"
+	kuadrantv1beta2 "github.com/kuadrant/kuadrant-operator/api/v1beta2"
+)
+
+var kuadrant_namespace = "kuadrant-system"
```

Comment:
I don't like the idea of  a hardcoded namespace, but I am not sure how we can best get the namespace the operator is installed into at the early stage when setting up the runables